### PR TITLE
SLE 2024-08 release notes

### PR DESCRIPTION
**Pull request by kaviarasu-ni**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/install-systemlink-enterprise/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Release notes for the 2024-08 release

### Why should this Pull Request be merged?

Required for release

### What testing has been done?

NA

### CODE REVIEW

**Comment by CIakab-NI** [comment]
@prestwick @vpstoynova , Could you help review before the end of this week: the new features & behavior changes, plus the helm chart breaking changes.
SL Squads have successfully validated the release and Customer Success is to validate this week.
I'll add the bugs, sbom and notices this week.

- For the documentation from Vera, please refer to [systemlink-enterprise-new-features-and-changes.md](https://dev.azure.com/ni/Users/_git/Documentation%20Reviews?path=%2Freview%20output%2Fsystemlink-enterprise%20md%2FSystemLink%20Enterprise%20New%20Features%20and%20Changes%2Fsystemlink-enterprise-new-features-and-changes.md&_a=preview)

- For the documentation from the squads, please refer to [SLE 2024-08_Helm Chart Breaking Changes, Software Behavior Changes (including Priviliges) for the Release Notes - out of July 2024.xlsx](https://nio365.sharepoint.com/:x:/s/ASWSystemLink/EfMEQ3OJB6JJgOq1Xk_4_lsB4lKudnkZJTgqbVWXhfq8_A?e=upUkBB)

File: release-notes/2024-08/README.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+## New Features and Behavior changes
+
+- Approve, delete, or disable multiple routines at once. Navigate to Analysis » Routines and select the routines you want to modify.
+- View and manage [data tables](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/visualizing-data-tables-in-a-data-space.html) across all test results. Navigate to Product Insights » Data Tables and select the data tables you want to interact with.
+- Learn about [data limits for proxy servers and for ingress controllers](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/data-limits-for-proxy-servers-ingress-controllers.html).
+- Learn about the [performance metrics for the SystemLink Alarm Service](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/performance-metrics-for-alarm-service.html).
+- Privileges "Create/Update/Delete Product and Test results" were split to "Create/Update/Delete Product" and "Create/Update/Delete Test Results".
+Existing Built-In Roles have been updated so that the roles with "Create/Update/Delete Product and Test results" get automatically the split privileges.
+Privileges for Steps are implicit set to the Test Result privileges.
```
Comment (vpstoynova) [inline release-notes/2024-08/README.md:13]:
```suggestion
Privileges for Steps are implicitly set to the Test Result privileges.
```

File: release-notes/2024-08/README.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+- Privilege "Access web application" was added under Data Tables category to show/hide the Data tables app in left navigation pane. This privilege is enabled by default for Collaborator, Data Maintainer and Systems Maintainer built-in roles.
```
Comment (vpstoynova) [inline release-notes/2024-08/README.md:14]:
```suggestion
- Privilege "Access web application" was added under Data Tables to show or hide the Data Tables application in left navigation pane. This privilege is enabled by default for Collaborator, Data Maintainer, and Systems Maintainer built-in roles.
```

File: release-notes/2024-08/README.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+- Privileges "Manage feeds" and "Apply states" were added under Systems Management category. They are enabled by default in Systems Maintainer built-in role.
```
Comment (vpstoynova) [inline release-notes/2024-08/README.md:15]:
```suggestion
- Privileges "Manage feeds" and "Apply states" were added under Systems Management. They are enabled by default in the Systems Maintainer built-in role.
```

**Review by prestwick — APPROVED** [review]

**Pull request merged**